cognite/client/utils/__init__.py

```diff
@@ -18,4 +19,12 @@
     else:
         from backports.zoneinfo import ZoneInfo
 
-__all__ = ["ZoneInfo", "MAX_TIMESTAMP_MS", "MIN_TIMESTAMP_MS", "datetime_to_ms", "ms_to_datetime", "timestamp_to_ms"]
+__all__ = [
+    "ZoneInfo",
+    "MAX_TIMESTAMP_MS",
+    "MIN_TIMESTAMP_MS",
+    "datetime_to_ms",
+    "ms_to_datetime",
+    "timestamp_to_ms",
+    "datetime_to_gql_timestamp",
```

Comment:
Let's also add this to the docs! A bonus would be to incorporate it in a docstring code example 🍾

Reply:
Done for the new function. Examples for the other functions is out of scope for this PR